### PR TITLE
Fix websocket examples

### PR DESCRIPTION
**Pull request by ricky0123**
Before submitting your PR, check that you've:

- [x] Bumped the version number

After submitting your PR:

- [x] Update the Changelog.md file with a link to your PR
- [ ] Check that CI passes (or if it fails, for reasons unrelated to your change, like CI timeouts)

### CODE REVIEW

**Comment by ricky0123** [comment]
I'm still a beginner with Haskell so I don't know why this is necessary, but it should resolve #1789.

File: yesod-test/ChangeLog.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # ChangeLog for yesod-test
 
+## 1.6.16
+
+* Fix yesod-websockets chat examples ([#1790](https://github.com/yesodweb/yesod/pull/1790)).
```
Comment (snoyberg) [inline yesod-test/ChangeLog.md:5]: I think this should appear in the yesod-websockets changelog, not the yesod-test changelog.

File: yesod-websockets/chat-with-multiple-channels.hs [hunk 2]
```diff
@@ -15,12 +15,12 @@ import UnliftIO.Exception (try, SomeException)
 
 data App = App (TVar (M.Map Text (TChan Text, Int)))
 
-instance Yesod App
-
 mkYesod "App" [parseRoutes|
 / HomeR GET
 |]
 
+instance Yesod App
```
Comment (snoyberg) [inline yesod-websockets/chat-with-multiple-channels.hs:22]: As I understand, there was a change in Template Haskell in recent GHC versions that made this change necessary. I don't know the details, sorry.

**Review by snoyberg — CHANGES_REQUESTED** [review]

**Review by snoyberg — APPROVED** [review]
Thanks!

**Pull request merged**